### PR TITLE
Gb/examples

### PR DESCRIPTION
**Pull request by grantbuster**

### CODE REVIEW

**Comment by grantbuster** [comment]
@ppinchuk you dont have to do a real review if you don't want, but this should be a good resource when people come asking about hsds servers.

File: examples/HSDS/README.rst [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+The easiest way to get started with HSDS is to get a developer API key via the
+`NREL Developer Network <https://developer.nrel.gov/signup/>`_. Once you have
+your API key, create an HSDS config file at ``~/.hscfg`` with the following
+entries (make sure you update the ``hs_api_key`` entry):
+
+.. code-block:: bash
+
+  # NREL dev api
+  hs_endpoint = https://developer.nrel.gov/api/hsds
+  hs_username =
```
Comment (ppinchuk) [inline examples/HSDS/README.rst:20]: Do they need to fill in their username/password too, or should they leave it blank like this?

**Review by ppinchuk — APPROVED** [review]
OMG thank you for this. I've definitely gotten questions about this and really didn't know how to help. This is an awesome addition!

File: examples/HSDS/README.rst (hunk 1 above)
Comment (grantbuster) [reply examples/HSDS/README.rst:20]: No need for un/pw. You actually don't need these lines, they're an artifact of using the hsconfig tool. Will remove.

**Pull request merged**